internal/rbd/rbd_util.go

```diff
@@ -117,6 +118,20 @@ type rbdImage struct {
 	ioctx *rados.IOContext
 }
 
+type logFileStrategy uint64
```

Comment:
what is the reason to use an `uint64` for this? it seems simpler to just have `logFileStrategy` as a `string`?

Reply:
yep, I was thinking in the similar line,  so we dont need the enum and also map there..